api/EXT_disjoint_timer_query.json

```diff
@@ -16,7 +16,7 @@
             "notes": "Removed due to the <a href='https://www.vusec.net/projects/glitch/'>GLitch exploit</a>."
           },
           "edge": {
-            "version_added": false
+            "version_added": "80"
```

Comment:
Are you sure about this? Did Edge decide to enable this extension, but Chrome still hasn't?

Reply:
Doing some more digging into this, it looks like it's enabled in macOS on both Chrome and Edge, but not on Windows. Doing some quick testing, it seems that it was removed due to the GLitch exploit as mentioned, and then added back only for macOS in Chromium 70. I'll update the data accordingly!